semgrep/tests/e2e/snapshots/test_ci/test_full_run/autofix-bitbucket/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+    "meta": {
+        "semgrep_version": "<sanitized version>",
+        "repository": "project_name/project_name",
+        "repo_url": "https://github.com/project_name/project_name.git",
+        "branch": "some/branch-name",
+        "ci_job_url": "https://github.com/project_name/project_name.git/addon/pipelines/home#!/results/a-uuid",
```

Comment:
This CI job URL doesn't look like it'd work, considering it's under github.com

Reply:
oop nice catch, fixed!